src/optimizer/statistics/operator/propagate_join.cpp

```diff
@@ -77,7 +77,7 @@ void StatisticsPropagator::PropagateStatistics(LogicalComparisonJoin &join, uniq
 				} else {
 					// this is the only condition and it is always true: all conditions are true
 					switch (join.join_type) {
-					case JoinType::SEMI: {
+					case JoinType::LEFT_SEMI: {
```

Comment:
Can we add cases for `RIGHT_SEMI` and `RIGHT_ANTI` here? The logic is exactly the same as for `LEFT_SEMI` and `LEFT_ANTI`, just flipped.